ccp/modules/exporter-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: "3.7"
+
+services:
+  exporter:
+    image: docker.verbis.dkfz.de/ccp/dktk-exporter:latest
+    container_name: bridgehead-ccp-exporter
+    environment:
+      LOG_LEVEL: "INFO"
+      EXPORTER_API_KEY: "${EXPORTER_API_KEY}"
+      CROSS_ORIGINS: "https://${HOST}"
+      EXPORTER_DB_USER: "exporter"
+      EXPORTER_DB_PASSWORD: "${EXPORTER_DB_PASSWORD}" # Set in exporter-setup.sh
+      EXPORTER_DB_URL: "jdbc:postgresql://exporter-db:5432/exporter"
+      CLEAN_TEMP_FILES_CRON_EXPRESSION: "0 0 1 * * *"
+      TEMP_FILES_LIFETIME_IN_DAYS: "1"
+      CLEAN_WRITE_FILES_CRON_EXPRESSION: "0 0 2 * * *"
+      WRITE_FILES_LIFETIME_IN_DAYS: "30"
+      HTTP_RELATIVE_PATH: "/ccp-exporter"
+      HTTP_SERVLET_REQUEST_SCHEME: "https"
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.exporter_ccp.rule=PathPrefix(`/ccp-exporter`)"
+      - "traefik.http.services.exporter_ccp.loadbalancer.server.port=8092"
+      - "traefik.http.routers.exporter_ccp.tls=true"
+      # TODO: Add basic auth in Teiler UI to access exporter?
+      #- "traefik.http.routers.exporter_ccp.middlewares=auth"
+      - "traefik.http.middlewares.exporter_ccp_strip.stripprefix.prefixes=/ccp-exporter"
+      - "traefik.http.routers.exporter_ccp.middlewares=exporter_ccp_strip"
+    volumes:
+      - "bridgehead-exporter:/app/exporter-files"
```

Comment:
Please check if you can do without this volume.

Reply:
See the answer to the previous comment.